packages/oak/lib/core/Editable/index.test.js

```diff
@@ -0,0 +1,164 @@
+import { fireEvent, render, waitFor } from '@testing-library/react';
+import { useRef } from 'react';
+import { withBuilder } from '@tests-utils';
+
+import Editable from '.';
+
+describe('<Editable />', () => {
+  const ComplexElement = ({
+    component = { settings: {} },
+    element = { type: {} },
+  }) => {
+    const containerRef = useRef();
+
+    const openMenu = () => {
+      containerRef.current?.toggle();
+    };
+
+    const closeMenu = () => {
+      containerRef.current?.close();
+    };
+
+    const toggleMenu = () => {
+      containerRef.current?.toggle();
+    };
+
+    return (
+      <Editable
+        component={component}
+        element={element}
+        ref={containerRef}
+      >
+        <div>
+          <button onClick={openMenu} id="open" />
+          <button onClick={closeMenu} id="close" />
+          <button onClick={toggleMenu} id="toggle" />
+        </div>
+      </Editable>
+    );
+  };
+
+  it('should render', () => {
+    const { container } = render(<Editable><p>hello</p></Editable>);
+    expect(container.innerHTML).toContain('hello');
+  });
+
+  it('should open menu when clicked', async () => {
+    const { container } = render(withBuilder(
+      <ComplexElement />, { getOverrides: () => jest.fn() }
+    ));
+    expect(container.querySelector('.oak-opened')).toBeNull();
+    fireEvent.click(container.querySelector('#open'));
+    await waitFor(
```

Comment:
always leave the arrow function on the upper line

```js
await waitFor(() => (
  expect(...).toBe(...)
));
```

and if you're on multiple lines, it's always better to wrap everything inside the immediately-returning arrow function in an additional `()`